ensek/client.py

```diff
@@ -6,16 +6,42 @@
 import stringcase
 import requests
 from requests.exceptions import RequestException
+from functools import wraps
+from tenacity import (
+    retry, before_log, wait_fixed, stop_after_attempt, retry_if_exception_type
+)
 
 logger = logging.getLogger(__name__)
 
+API_CALL_RETRIES = 5
+API_CALL_RETRY_WAIT = 1 
```

Comment:
can you add these as __init__ options, default retries to 0, and if its 0, dont retry at all